src/System.Net.Requests/tests/HttpRequestResponseSerializationTest.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.Serialization.Formatters.Tests;
+using Xunit;
+
+namespace System.Net.HttpRequest.Tests
+{
+    public partial class HttpRequestResponseSerialization
+    {
+        public readonly static object[][] EchoServers = System.Net.Test.Common.Configuration.Http.EchoServers;
+
+        [OuterLoop]
+        [Theory, MemberData(nameof(EchoServers))]
```

Comment:
Nit: we don't need to run this test with both http and https... we're not actually looking at the networking aspects of this and rather just that the object can be properly serialized/deserialized.